### DISCUSSION TITLE
Droppare asciidoc-thesis

### DISCUSSION THREAD

**Discussion by VaiTon** [csunibo/csunibo.github.io#54] [Sondaggi]
Asciidoc è veramente troppo scarno per poterci scrivere una tesi. Non so quanto abbia senso mantenere il template per farlo.

**Comment by foxyseta** [comment]
Ho votato "tieni" perché anche se al momento lo sviluppo si è fermato (l'unico sviluppatore era @cristian-5), non ci è mai costato nulla lasciarla non archiviata. Anche se non ho mai usato Asciidoc, [guardando la documentazione](https://docs.asciidoctor.org/asciidoc/latest/) mi sembra le funzionalità per una buona tesi triennale/magistrale ci siano tutte.

Penso che lasciare non-archiviato rimuova un ostacolo (anche solo mentale) a future nuove persone contribuenti. Se non ce ne saranno, non ci avremo perso nulla.

Se alla fine si decide di archiviare, ricordiamoci di segnare come `draft: true` la relativa pagina wiki, e rimuovere il progetto dalla sezione "Tesi" della pagina principale di CartaBinaria.
